src/RcppExports.cpp

```diff
@@ -1,6 +1,7 @@
 // Generated by using Rcpp::compileAttributes() -> do not edit by hand
 // Generator token: 10BE3573-1514-4C36-9D1C-5A225CD40393
 
+#include <RcppEigen.h>
```

Comment:
So my Rcpp was removing this, not really sure why yours would be adding that. We're not using any of the RcppEigen features so think this will just inflate the compiled code a bit but not a big deal.